reconcile/templating/renderer.py

```diff
@@ -269,9 +269,9 @@ def reconcile_template_collection(
         gql_api: GqlApi,
         persistence: FilePersistence,
         ruamel_instance: yaml.YAML,
-        input: TemplateInput,
+        result: TemplateResult,
         each: dict[str, Any],
-    ) -> list[TemplateOutput]:
+    ) -> None:
```

Comment:
the old signature is easier to understand, can we remove the `result` from params, and still return `list[TemplateOutput]`?